dids/bearerdid_test.go

```diff
@@ -21,7 +21,7 @@ func Test_ToKeys(t *testing.T) {
 
 	vm := portableDID.VerificationMethod[0]
 
-	assert.NotEqual[jwk.JWK](t, vm.PublicKeyJWK, jwk.JWK{}, "expected publicKeyJwk to not be empty")
```

Comment:
FYI the explicit parametric types should be unnecessary here.

Reply:
I'm a little new to generics and seems like every time I use one of these functions vscode adds [] for me. figured it's a thing?

Reply:
Ah. No that's just VSCode being _very_ over enthusiastic.